package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "supertokens-node",
```

Comment:
missing tests..?
- Should not log if debug is undefined or false
- Should log if debug is set to true
- Should log if env var is set